x/kavamint/types/params.go

```diff
@@ -0,0 +1,87 @@
+package types
+
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter keys and default values
+var (
+	KeyCommunityPoolInflation = []byte("CommunityPoolInflation")
+	KeyStakingRewardsApy      = []byte("StakingRewardsApy")
+
+	// default inflation values are zero
+	DefaultCommunityPoolInflation = sdk.ZeroDec()
+	DefaultStakingRewardsApy      = sdk.ZeroDec()
+
+	// MaxMintingRate returns the per second rate equivalent to 10,000% per year
+	MaxMintingRate = sdk.NewDec(100)
+)
+
+// NewParams returns new Params with inflation rates set
+func NewParams(communityPoolInflation sdk.Dec, stakingRewardsApy sdk.Dec) Params {
```

Comment:
Feels like we could have gone with a more general approach controllable by governance, though just food for thought -- not saying we should make these changes.

```
Params:
  Minters:
    - name: "staking"
       source_module_account: "bonded_pool"
       destination_module_account: "fee_collector"
       last_rate:  "0.5"
       last_accumulation_time: "...."
  - name: "community_pool"
       source_module_account: "community"
       destination_module_account: "community"
       last_rate:  "0.4"
       last_accumulation_time: "...."
```

Reply:
definitely. i like that abstraction. we discussed this in the initial design meeting and decided that the implementation functions should accept a more general representation:
```go
func (k *Keeper) AccumulateInflation(
	ctx sdk.Context,
	rate sdk.Dec,
	basis sdk.Int,
	secondsSinceLastMint float64,
)
```
but decided it was safer overall to harden the params themselves by making each supported inflation rate specific.

when i refactor the begin blocker i can look into making the mints even more general by including source and destination like you have there.